### DISCUSSION TITLE
Usar whatsapp para chatear

### DISCUSSION THREAD

**Discussion by CesarAugusto316** [cometa-dani/cometa-mobile-app-ui#2] [Ideas]
bla bla bla

**Comment by dani97munoz** [comment]
Tengo una idea!!!! en la lista de matches se podra solicitar follow back para poder conseguir el numero de whatsapp de la otra persona. Pero, antes podra abrir el usuario un chat grupal especifico para cada evento con todas las personas que tambien quieran ir al evento. asi tambien se considera una app de red social y no solo para 1:1.
